pkg/image/importer/importer.go

```diff
@@ -299,9 +314,41 @@ func formatRepositoryError(repository *importRepository, refName string, refID s
 	return
 }
 
+func calculateImageSize(ctx gocontext.Context, repo distribution.Repository, image *api.Image, cache *imageStreamImporterCache) error {
```

Comment:
Add Godoc indicating why this method is necessary, i.e. for v2 schemas converted to v1 which have no stat info.  Also include a description of what an error means (we couldn't calculate the sizes) and why we have to exit out, as opposed to continuing (because quota may be on).